src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -99,8 +103,8 @@ const EventEntries = React.createClass({
         <EventTags
           group={group}
           event={evt}
-          orgId={orgId}
-          projectId={projectId} />
+          orgId={this.props.orgId}
+          projectId={project.slug} />
```

Comment:
my goal when writing it was that we could transition without renaming, but I don tknow if that'll ever happen (e.g. ideally the project ID *is* the slug)

it's also orgId and projectId everywhere, so def not renaming as part of this